src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -1281,6 +1281,28 @@ public void Start_HasStandardErrorEncodingNonRedirected_ThrowsInvalidOperationEx
             Assert.Throws<InvalidOperationException>(() => process.Start());
         }
 
+        [PlatformSpecific(TestPlatforms.Windows)]
+        [Fact]
+        public void Start_RedirectStandardOutput_StartAgainWithoutRedirect()
+        {
+            var process = new Process
+            {
+                StartInfo = new ProcessStartInfo
+                {
+                    FileName = "cmd",
+                    Arguments = "/C dir",
```

Comment:
We have helpers for starting processes, used elsewhere in the project, that don't require tying this to specific commands that only work on Windows. It's bee better to use those and make this test more general.